src/views/home.njk

```diff
@@ -1,8 +1,5 @@
-{% extends 'base.njk' %}
+{% extends 'common/base.njk' %}
```

Comment:
By moving base into common folder, I think we will have an issue with other pages (form pages for example).
Can you check with Sophia? Maybe it's better to wait before to change the structure of the folders because we have 2 other pull requests waiting to be merged.

Reply:
Yeah absolutely, I can sync up with Sophia - It would be a good idea to amend the structure at this point as there are now two common njk files now, which follows the pattern as other pod repos - so makes sense at this point? The only place i can foresee a merge conflict would be the link back to the base file (which i've amended) unless you can think of any others?

`{% extends 'common/base.njk' %}` so the merge conflict would be accepted at the point of rebase

Reply:
It's not a very hard conflict to resolve, so up to you and sophia to be well synchronised.